embiggen.gemspec

```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |s|
   s.name = 'embiggen'
-  s.version = '1.7.0'
+  s.version = '1.7.1'
```

Comment:
In the past, adding a new shortener resulted in a minor version bump. I'd follow the same convention

Reply:
Done